desktop/hardened-desktop/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hardened Desktop
+description: Overview of what Hardened Desktop is
+keywords: security, hardened desktop, enhanced container isolation, registry access management, admin controls, root access, administrators, docker desktop
+---
+>Note
+>
+>Hardened Desktop is available to Docker Business customers only.
+
+Hardened Desktop is a new security model for Docker Desktop. It's part of Docker's ongoing effort to increase Docker Desktop security without impacting the developer experience.
```

Comment:
I'd change to:

"Hardened Desktop is a new security model for Docker Desktop which is designed to provide admins with a simple and powerful way to improve their organizations security posture for containerised development, without impacting the great developer experience that Docker Desktop offers."

let's immediately convey the value of Hardened Desktop and who it's for. 'ongoing effort to increase security' sounds like we're going through a bit of a slog.